projects/igniteui-angular/src/lib/grids/tree-grid/tree-cell.component.html

```diff
@@ -149,7 +149,8 @@
     <ng-container *ngIf="level > 0">
         <div
             #indentationDiv
-            class="igx-grid__tree-cell--padding-level-{{ level }}"
+            class="igx-grid__tree-cell--padding-level-{{level}}"
+            style="{{'--igx-tree-indent-level:' + level}}; padding-inline-start: calc(var(--igx-tree-indent-size) * var(--igx-tree-indent-level))"
```

Comment:
Do we need to declare a separate variable like `--igx-tree-indent-level`? Can't we just use the value directly in the calc function?

Reply:
I tried different ways, but every time I get an error. For some reason, the interpolation {{level}} is not working inside the calc function. Do you have any idea how to make it work, I agree that this would be ideal but it does not work 
`padding-inline-start: calc(var(--igx-tree-indent-size) * {{level}})`